src/app/layout.tsx

```
@@ -122,22 +123,67 @@ export const metadata: Metadata = {
+    publisher: {
+      "@type": "Organization",
+      name: "Rapid AI",
+    },
+  };
+
   return (
     <html lang="en">
+      <head>
+        <script
+          type="application/ld+json"
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)